ebml_matroska.xml

```diff
@@ -1215,7 +1215,8 @@ the bytes that were removed from the beginning of each frames of the track.</doc
   </element>
   <element name="ContentEncryption" path="\Segment\Tracks\TrackEntry\ContentEncodings\ContentEncoding\ContentEncryption" id="0x5035" type="master" maxOccurs="1">
     <documentation lang="en" purpose="definition">Settings describing the encryption used.
-This Element **MUST** be present if the value of `ContentEncodingType` is 1 (encryption) and **MUST** be ignored otherwise.</documentation>
+This Element **MUST** be present if the value of `ContentEncodingType` is 1 (encryption) and **MUST** be ignored otherwise.
+Decoding implementations **MAY** support encryption.</documentation>
```

Comment:
Same remark, this should be `Matroska Readers`.

Reply:
I changed to `Matroska Player` as a `Matroska Reader` does not need to handle that (transporting the opaque packets, splitting them, etc, does not require the support of this feature).